Art/cTahirih/Wave by Paul Salentiny remastered by Andres.css

```diff
@@ -0,0 +1,524 @@
+
+body {
+	background:  linear-gradient(#ffffff 50%, rgba(255,255,255,0) 0) 0 0,
+	radial-gradient(circle closest-side, #FFFFFF 53%, rgba(255,255,255,0) 0) 0 0,
+	radial-gradient(circle closest-side, #FFFFFF 50%, rgba(255,255,255,0) 0) 55px 0 #48B;
+	background-size: 110px 200px;
+	background-repeat: repeat-x;
+	width: 100vw;
+	height: 100vh;
+}
+
+.container {
+	position: absolute;
+	top: 20%;
+	left: 25%;
+	width: 500px;
+	height: 700px;
```

Comment:
Could you please delete this file from cTahirih? I commit it without wanting it to happen. Thank you.

Reply:
@royranger Hey, I hope you could help me delete this commit that was unintentionally uploaded. Thanks.